src/axel.c

```diff
@@ -152,6 +152,26 @@ axel_t *axel_new( conf_t *conf, int count, const void *url )
 	if( ( s = strchr( axel->filename, '?' ) ) != NULL && axel->conf->strip_cgi_parameters )
 		*s = 0;		/* Get rid of CGI parameters */
 
+	if( axel->conf->no_clobber && access( axel->filename, F_OK ) == 0 )
+	{
+		char stfile[MAX_STRING + 3];
+
+		sprintf( stfile, "%s.st", axel->filename );
+		if( access( stfile, F_OK ) == 0 )
+		{
+			printf( _("Ignoring no-clobber option because the "
```

Comment:
This message could be `"Incomplete download found, ignoring no-clobber option\n"`.